src/Data/TypeLevel/Num/Ops.hs

```diff
@@ -386,7 +386,7 @@ infixr 8 ^
 -- @log_base_b x = e@
 --  Note it is not relational (i.e. cannot be used to express exponentiation)
 class (Pos b, b :>=: D2, Pos x, Nat e) =>  LogBase b x e  | b x -> e 
-instance  LogBaseF b x e f => LogBase b x e
+instance  (Pos b, b :>=: D2, Pos x, Nat e, LogBaseF b x e f) => LogBase b x e
```

Comment:
Yeah, it seems so ([Migration/8.0](https://ghc.haskell.org/trac/ghc/wiki/Migration/8.0)). Thanks for answering so quick!

Check the output for Job #5.2 here [![Build Status](https://travis-ci.org/ugeorge/type-level.svg?branch=master)](https://travis-ci.org/ugeorge/type-level)

It complains heavily along the lines of:

    src/Data/TypeLevel/Num/Ops.hs:375:10: warning: [-Wredundant-constraints]
        • Redundant constraint: ExpBase b D7 r
        • In the instance declaration for ‘ExpBase b D8 r'’

As I said, these warning could just be repressed. On the other hand I cannot estimate the effort in trying to deal with them. Would taking out these "redundant" constraints mess up (too much) the state of the library?